lib/manager/python/package.js

```diff
@@ -0,0 +1,33 @@
+
+const got = require('got');
+const { isGreaterThan, semverSort } = require('../../util/semver');
```

Comment:
We will likely need to write our own python version utils in parallel to semver, because the syntax isn't identical. If we start with exact python versions only to begin then we can probably leverage semver like this.

Reply:
Even `semver` won't handle core semantic version requirements, such as [Epochs](https://www.python.org/dev/peps/pep-0440/#version-epochs). So I definitely think we'll need to investigate a Python-compatible semantic version library early on.